frappe/tests/ui/test_kanban/test_kanban_filters.js

```diff
@@ -0,0 +1,34 @@
+QUnit.module('views');
+
+QUnit.test("Test filters [Kanban view]", function(assert) {
+	assert.expect(2);
+	let done = assert.async();
+
+	frappe.run_serially([
+		() => frappe.tests.setup_doctype('User'),
```

Comment:
We don't need `setup_doctype` now, I think. The new tests are run in order specified by **tests.txt**